diff --git ecmcApp/src/drvAsynECMCPort.cpp ecmcApp/src/drvAsynECMCPort.cpp
index 2c3c46c..d68bc11 100644
--- ecmcApp/src/drvAsynECMCPort.cpp
+++ ecmcApp/src/drvAsynECMCPort.cpp
@@ -299,10 +299,10 @@ drvAsynECMCPortConfigure(const char *portName,
   ecmcController_p->octet.drvPvt = ecmcController_p;
   status = pasynOctetBase->initialize(ecmcController_p->portName,
                                       &ecmcController_p->octet, 0, 0, 1);
-  printf("%s/%s:%d XXXX version=%s %s\n",
-         __FILE__, __FUNCTION__, __LINE__,
-         git_version_str,
-         portName ? portName : "");
+  // printf("%s/%s:%d XXXX version=%s %s\n",
+  //        __FILE__, __FUNCTION__, __LINE__,
+  //        git_version_str,
+  //        portName ? portName : "");
   if(status != asynSuccess) {
     printf("EEEEEEEEE drvAsynECMCPortConfigure: pasynOctetBase->initialize failed.\n");
     ecmcController_pCleanup(ecmcController_p);
